modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductConfigurationResourceImpl.java

```diff
@@ -307,9 +307,7 @@ public ProductConfiguration patchProductConfiguration(
 				GetterUtil.getBoolean(
 					!_isTaxable(productTaxConfiguration),
 					cpConfigurationEntry.isTaxExempt()),
-				GetterUtil.getBoolean(
-					productConfiguration.getVisible(),
-					cpConfigurationEntry.isVisible()),
+				true,
```

Comment:
but these should not be needed anymore, why are we keeping this parameter in the sign of the method?